asyncapi/asgardeo/service_types.bal

```diff
@@ -14,6 +14,7 @@ public type UserOperationService service object {
 
 public type LoginService service object {
     remote function onLoginSuccess(LoginSuccessEvent event) returns error?;
+    remote function onLoginFailed(LoginFailedEvent event) returns error?;
```

Comment:
Add a comment quite similar to https://github.com/ballerina-platform/asyncapi-triggers/blob/ead5cbc9011c7d56af17761e4d45f8b5cb2f78d0/asyncapi/shopify/service_types.bal#L46

Reply:
Resoloved in [e8d6609](https://github.com/ballerina-platform/asyncapi-triggers/pull/154/commits/e8d660936bbc0a411944c843eb7e01bbc72d117c)